docs/process.md

```diff
@@ -0,0 +1,41 @@
+# Development Process
+
+Notes:
+* The development process is a simplified version of the one used in [TEAMMATES](https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md). 
+* The `master` branch always contains the latest stable code (may contain unreleased features/fixes).
+* The `release` branch always contains the latest released version.
```

Comment:
Okay, so I'll merge into `rc` and do a public release for last week's version today. This week's version will go into `release` next week.

So the new workflow for releasing is:

- Initial state: `master` is ahead of `rc` and `release`
- Bump version numbers, commit to master with version number as message
- Merge `master` into `rc`, same commit message
- One week later, merge `rc` into `release`, same commit message
- Tag `release`; uploaded jar will be associated with this tag

No more cherry-picking (causes merge conflicts), and we tag only when we are about to release.